pkg/install/helm/cli.go

```diff
@@ -171,3 +175,17 @@ func (c *cli) run(namespace string, args ...string) ([]byte, error) {
 
 	return stdoutStderr, err
 }
+
+func valuesToFlags(values map[string]string) []string {
+	set := make([]string, 0)
```

Comment:
Small thing, as we know already the size of the map you can use instead:
```
make([]string, 0, len(values))
```
to avoid allocations.